### PR TITLE
Fix: Missing Help Component Graphic

### PR DESCRIPTION
**Pull request by omgaz**
Was previously not exporting the help icon.
Also fixed panel titles to work with help icon.

### CODE REVIEW

File: src/components/adslotUi/HelpIconPopoverComponent.jsx [hunk 1]
```diff
@@ -1,21 +1,19 @@
 import React, { PropTypes } from 'react';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Popover from 'react-bootstrap/lib/Popover';
-import SvgSymbol from 'components/alexandria/SvgSymbolComponent';
 import expandDts from '../../helpers/expandDtsHelper';
 
 require('styles/adslotUi/HelpIconPopover.scss');
```
Comment (sonhanguyen) [inline src/components/adslotUi/HelpIconPopoverComponent.jsx]: can we change this to `import` for consistency, or it needs to stay like that for some reason?

**Comment by sonhanguyen** [comment]
LGTM

File: src/components/adslotUi/HelpIconPopoverComponent.jsx (hunk 1 above)
Comment (omgaz) [inline src/components/adslotUi/HelpIconPopoverComponent.jsx]: All of these are using `require`

Comment (patrick-ivers) [inline src/components/adslotUi/HelpIconPopoverComponent.jsx]: @sonhanguyen Not sure why, but all the CSS stuff is imported using `require`. We could look at changing this in a later refactor.

**Comment by patrick-ivers** [comment]
Merging...

**Pull request merged**